quick_tour/flex_recipes.rst

```diff
@@ -0,0 +1,249 @@
+Flex: Compose your Application
+==============================
+
+After reading the first part of this tutorial, you have decided that Symfony was
+worth another 10 minutes. Great choice! In this second part, you'll learn about
+Symfony Flex: the amazing tool that makes adding new features as simple as running
+one command. It's also the reason why Symfony is perfect for a small micro-service
+or a huge application. Curious? Perfect!
+
+Symfony: Start Micro!
+---------------------
+
+Unless you're building a pure API (more on that soon!), you'll probably want to
+render HTML. To do that, you'll use `Twig`_. Twig is a flexible, fast, and secure
+template engine for PHP. It makes your templates more readable and concise; it also
+makes them more friendly for web designers.
+
+Is Twig already installed in our application? Actually, not yet! And that's great!
+When you start a new Symfony project, it's *small*:  only the most critical dependencies
+are included in your ``composer.json`` file:
+
+.. code-block:: text
+
+    "require": {
+        "...",
+        "symfony/console": "^4.1",
+        "symfony/flex": "^1.0",
+        "symfony/framework-bundle": "^4.1",
+        "symfony/yaml": "^4.1"
+    }
+
+This makes Symfony different than any other PHP framework! Instead of starting with
+a *bulky* app with *every* possible feature you might ever need, a Symfony app is
+small, simple and *fast*. And you're in total control of what you add.
+
+Flex Recipes and Aliases
+------------------------
+
+So how can we install and configure Twig? Just run one command:
+
+.. code-block:: terminal
+
+    $ composer require twig
+
+Two *very* interesting things happen behind the scenes thanks to Symfony Flex: a
+Composer plugin that is already installed in our project.
+
+First, ``twig`` is not the name of a Composer package: it's a Flex *alias* that
+points to ``symfony/twig-bundle``. Flex resolves that alias for Composer.
+
+And second, Flex installs a *recipe* for ``symfony/twig-bundle``. What's a recipe?
+It's a way for a library to automatically configure itself by adding and modifying
+files. Thanks to recipes, adding features is seamless and automated: install a package
+and you're done!
+
+You an find a full list of recipes and aliases by going to `https://symfony.sh`_.
+
+What did this recipe do? In addition to automatically enabling the bundle in
```

Comment:
*"enabling the bundle"* <-- will people understand that? It's the first time that "bundle" is mentioned in the Quick Tour. Maybe: *"enabling the feature"* is better?